order-service/order-service.ts

```diff
@@ -0,0 +1,184 @@
+import { Tx, Pair, Position, OrderConfig } from '../types/common.js';
+import {
+  OrderSide,
+  OrderType,
+  SubaccountClient,
+  Network,
+  CompositeClient,
+  IndexerClient,
+  BECH32_PREFIX,
+} from '@dydxprotocol/v4-client-js';
+
+import v4_client_js from '@dydxprotocol/v4-client-js';
```

Comment:
- Don't use `snake_case`, it's not a convention in JS. The best option for this package import, which includes other objects, is probably `PascalCase`.
- You can merge the two import statements (`import defExp, { namedExp } from 'some-package';`)